view/index.erb

```diff
@@ -0,0 +1,10 @@
+lets peep!
+<div>
+  <% if @tweets.length %>
+    <ul>
+      
+      <li><%= @tweet %></li>
```

Comment:
Nice, totally on the right lines here. What do you see if you run your app manually? 

I think if you want to display each tweet, would need to iterate over the `@tweets` variable. Similar to this: 
https://github.com/makersacademy/mvc-example/blob/master/views/books.erb#L13